gtsam/linear/NoiseModel.h

```diff
@@ -924,7 +890,8 @@ namespace gtsam {
 
         GemanMcClure(double c = 1.0, const ReweightScheme reweight = Block);
         virtual ~GemanMcClure() {}
-        virtual double weight(double error) const;
+        virtual double weight(const double error) const;
```

Comment:
@dellaert what do you have to say about these `virtual` identifiers? Would we need to subclass the estimators?

Reply:
Good point: the base class should have virtual, but derived classes should have override in c++11

Reply:
Doesn't this mean all the estimators should use `override`?

Reply:
Yes

Reply:
All done.